tests/conftest.py

```diff
@@ -361,7 +361,7 @@ def default_ctr_kwargs(ctr_mgr: CtrMgr, setup_mode: Optional[str]) -> dict[str,
         kwargs["systemd"] = "always"
     else:
         # Systemd expects /run to be a tmpfs.
-        kwargs["tmpfs"] = ["/run"]
+        kwargs["tmpfs"] = ["/run", "/run/lock"]
```

Comment:
Why is this needed? If `/run` is a tmpfs anyway then `/run/lock` is at least within a tmpfs. Is there systemd documentation stating this requirement?

Reply:
For some reason, it just doesn't seem to create the folder but just assumes it exists.

Reply:
Systemd might be trying to create a tmpfs mount at `/run/lock` if it doesn't exist, which would fail if there's no `CAP_SYS_ADMIN`. Is that the problem? I'm still not sure *why* systemd feels the need for this to exist.